.travis.yml

```diff
@@ -3,12 +3,21 @@ sudo: required
 services:
   - docker
 
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
+script:
+  - docker-compose -f docker-compose-travis.yml build nearby-services-api-tests
```

Comment:
Think you want `nearby-services-api-tests` to be `web-app-test` (same for line below)